docs/components/icons/index.html

```diff
@@ -37,22 +38,54 @@ <h2 id="demo">Demo</h2>
     {% endcode %}
   </section>
 
-  <section>
-    <h2 id="available-icons">Available Icons</h2>
-    <p>There are {{ data.icons.length }} icons available!</p>
-    <div class="hxRow">
-      {% for icon in data.icons %}
-        <div class="hxCol hxSpan-12-xs hxSpan-6-sm hxSpan-4-md hxSpan-3-xl">
-          <div class="demo icon-demo">
-            <hx-icon type="{{icon}}"></hx-icon>
-            <p>
-              <code>{{icon}}</code>
-            </p>
-          </div>
+  <section id="vue-iconDemo">
```

Comment:
Rearranged the list of icons to use a "media" object pattern, so that the version for each icon can be explicitly stated.

![screen shot 2018-03-19 at 7 59 11 pm](https://user-images.githubusercontent.com/545605/37630013-02d580d2-2bb0-11e8-8d0f-06545aa1c32e.png)

Reply:
Tried using a tooltip. It doesn't feel right.

* No matter what direction the tooltip opens, it'll be covering up other information.
* The stripped down version metadata seems lacking, too.

![screen shot 2018-03-29 at 2 11 58 pm 2](https://user-images.githubusercontent.com/545605/38109386-9596c7dc-335d-11e8-915f-f0f3053851d3.png)